etc/weblate/settings.py

```diff
@@ -256,6 +256,15 @@
 SOCIAL_AUTH_GITLAB_SECRET = os.environ.get('WEBLATE_SOCIAL_AUTH_GITLAB_SECRET', '')
 SOCIAL_AUTH_GITLAB_SCOPE = ['read_user']
 
+if 'WEBLATE_SOCIAL_AUTH_AUTH0_KEY' in os.environ:
+    AUTHENTICATION_BACKENDS += ('social_core.backends.auth0.Auth0OAuth2',)
+    SOCIAL_AUTH_AUTH0_SCOPE = ['openid', 'profile', 'email']
+
+if 'WEBLATE_SOCIAL_AUTH_AUTH0_AUTH_EXTRA_ARGUMENTS' in os.environ:
+    auth0_args = get_env_map('WEBLATE_SOCIAL_AUTH_AUTH0_AUTH_EXTRA_ARGUMENTS')
+    SOCIAL_AUTH_AUTH0_AUTH_EXTRA_ARGUMENTS = auth0_args
+    os.environ.pop('WEBLATE_SOCIAL_AUTH_AUTH0_AUTH_EXTRA_ARGUMENTS')
```

Comment:
I'd simplify all three lines to only:

```suggestion
    SOCIAL_AUTH_AUTH0_AUTH_EXTRA_ARGUMENTS = get_env_map('WEBLATE_SOCIAL_AUTH_AUTH0_AUTH_EXTRA_ARGUMENTS')
```

Also I see no reason for `pop`.

Reply:
Right, this was from openshift config, where at the end there was a function of replacing all `WEBLATE_`, so this would get overwritten. Will remove it.